tsconfig.json

```diff
@@ -0,0 +1,52 @@
+{
+    "compilerOptions": {
+      "noEmit": true,
+      "target": "ES2015",
+      "lib": [
+        "dom",
+        "dom.iterable",
+        "esnext"
```

Comment:
Do we need esnext?

Reply:
We're down-leveling to a lower target, but TS let's us use modern features while writing and compiles them down to a more widely compatible version.

Frankly, though, I'm not sure what types this provides. I've always used it in my configs, though 🤷‍♂️

Basically, IDK.

Reply:
Funny thing - reading over https://www.typescriptlang.org/tsconfig#lib, it would seem that I'm right about the above - however 'dom' and 'dom.iterable' are pointless, since this never runs in the browser. I'll remove those two.